node-reinstall

```diff
@@ -193,17 +193,14 @@ rm -rf $HOME/{local,lib,include,node*,npm,.npm*,.node-gyp}
 sudo rm -rf $PREFIX/lib/node*
 sudo rm -rf $PREFIX/include/node*
 sudo rm -rf $PREFIX/bin/{node,npm}
+sudo rm -rf $PREFIX/share/doc/node
```

Comment:
These directories getting deleted should be added to the README in the block showing what gets deleted

Reply:
Done